src/Service/SaveIllustration.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Service;
+
+class SaveIllustration{
+
+    public function save($illustration, $photoDir) {
```

Comment:
can you make it more generic ? it could work with any illustration (type hint the parameter)